packages/cadl-ranch-specs/http/azure/core/page/main.tsp

```diff
@@ -200,3 +201,411 @@ interface TwoModelsAsPageItem {
   @route("/second-item")
   listSecondItem is global.Azure.Core.Foundations.Operation<{}, global.Azure.Core.Page<SecondItem>>;
 }
+
+@resource("transactions")
+model LedgerEntry {
```

Comment:
could we generalize the name of the models here so that it does not look like being borrowed from a service?

Reply:
I think the `@resource` decorator is the place that makes this case unique from others?
If so let us keep this.
If not, we should not add irrelevant stuff on cadl ranch cases.

Reply:
It is under the `azure` folder so I think it is more appropriate to keep the `@resource`.